jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/StandardDescriptorProcessor.java

```diff
@@ -176,7 +176,7 @@ public void visitContextParam(WebAppContext context, Descriptor descriptor, XmlP
                 if (descriptor instanceof FragmentDescriptor)
                 {
                     if (!((String)context.getInitParams().get(name)).equals(value))
-                        throw new IllegalStateException("Conflicting context-param " + name + "=" + value + " in " + descriptor.getResource());
+                        throw new IllegalStateException("Conflicting context-param " + name + "=" + value + " in " + descriptor);
```

Comment:
I'd use `descriptor.getURI()` here, since it's better than having in the exception message the descriptor's `toString()` which contains className, hashCode, etc.
See ee10's `MetaData.addFragmentDescriptor(...)`.

Reply:
ok, but the Descriptor.toString() won't contain classname / hashcode / etc.
It's just the simple name + uri already.